docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+# Turn The Senior Around
+
+As engineering managers we take many considerations into account, two of the main ones are:
+- Delivering on time
+- Make our people grow
+
+During my experience as an engineering manager and as an engineer all the managers I have met saw those subjects critical and invested a lot in improving them.<br>
+Another consensus among them all was that senior engineers are a key enabler to both topics, they are the main players the team lead can rely on to achieve those goals.<br>
+I spoke a bit about the role of senior engineer in my [The Last Dance blog](/docs/2020-07-09-last-dance) and this blog relates to it - I want to discuss about __*how*__ should engineering managers utilize their senior engineers to achieve those goals.
+
+Most of the engineering managers I have met used their senior engineers to lead all the critical features, I would like to turn the senior around and suggest the opposite - the more senior you are the the less you lead engineering efforts in the team.
+The more senior you are the more you promote other engineers by giving constant feedback on features __*they lead*__.
+
+Needless to say this only reflects my opinion :)
+
+# The Common Team Structure - Senior First Delivery
+![The common team structure](/docs/assets/turn-the-senior-around/common_way.png)<br><br>
+The common team structure I saw was senior engineers leading features, while trying to improve and grow the non senior members during the process.
+
+## How Does This Work
+The senior engineers are the ones responsible to make the feature delivered, they are the ones with all the context and they are the active leaders of those feature squads - communication with stakeholders, design etc.<br>
+The team lead's job is to make sure the senior engineers are on track and help them if needed.<br>
+The advantages are obvious: this way there is maximal certainty that the features will be delivered on time, the non senior members learn by watching the senior in action and by (hopefully) active mentoring.<br>
```

Comment:
I like using `advantages` to describe things that work well by choosing this team structure. For consistency, I'd replace the use of `issues` with `disadvantages`.

Another way to go here would have been using the terms `pros` & `cons`